converter/types.go

```diff
@@ -112,6 +119,13 @@ func prepareOCIBundleConfig(mediaType string) bundleConfigPreparer {
 func prepareNonOCIBundleConfig(blob []byte) (*PreparedBundleConfig, error) {
 	man, err := schema2.FromStruct(schema2.Manifest{
 		Versioned: schema2.SchemaVersion,
+		Layers: []distribution.Descriptor{
```

Comment:
I think you can factorize with the Config descriptor just below 👍

Reply:
Done